worker/games.py

```diff
@@ -1312,7 +1313,8 @@ def make_player(arg):
         tc_limit *= 2
 
     while games_remaining > 0:
-        batch_size = games_concurrency * 4  # update frequency
+        # update frequency: every 4 games at LTC, or a similar time interval at other TCs
+        batch_size = games_concurrency * 4 * max(1, round(tc_limit_ltc / tc_limit))
```

Comment:
the factor works only for TC < LTC, if it's intended better to improve the comment
```python
# update frequency: every 4 games at LTC, or a similar time interval at lower TCs
```